easybuild/easyconfigs/s/SPAdes/SPAdes-4.1.0-GCCcore-13.3.0.eb

```diff
@@ -0,0 +1,49 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+# Modified by:
+# Adam Huffman
+# The Francis Crick Institute
+# Updated by:
+# Filip Kružík (INUITS)
+# Pavel Tománek (INUIS)
+
+easyblock = 'CMakeMake'
+
+name = 'SPAdes'
+version = '4.1.0'
+
+homepage = 'https://github.com/ablab/spades'
+description = "Genome assembler for single-cell and isolates data sets"
+
+toolchain = {'name': 'GCCcore', 'version': '13.3.0'}
```

Comment:
@pavelToman moved to `GCCcore` because you don't think it needs to be at the `GCC` level?

Reply:
I don't see a reason to use `GCCcore` as toolchain for SPAdes, so changed is back to `GCC` in 30dacb1dc5